apps/examples/board_specific_demo/ameba_secure_world_demo/README.txt

```diff
@@ -0,0 +1,13 @@
+examples/secure_world
+^^^^^^^^^^^^^^
+
+  Secure World APIs is used to check when we access to read or to write in non-secure world, 
```

Comment:
We are currently not using any APIs, so please change the description. I think something like "This example depicts how the access from non secure to secure world is restricted. When non secure world tried to access(write/read) from the secure world region, it causes a secure fault"